testing/src/main/java/org/oppia/testing/FakeCrashLogger.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.testing
+
+import java.lang.Exception
+import org.oppia.testing.TestLogReportingModule.Companion.exceptionList
+import org.oppia.util.firebase.CrashLogger
+
+/**
+ * A test specific fake crash logger that adds the logged exception to a list of exceptions.
+ */
+class FakeCrashLogger : CrashLogger {
```

Comment:
Although it's trivial, I suggest adding a very basic test suite to verify the contract of the fake crash logger itself.

Reply:
Done.